packages/loader/container-loader/src/containerContext.ts

```diff
@@ -260,79 +132,15 @@ export class ContainerContext implements IContainerContext {
 		}
 		this._disposed = true;
 
-		this.lifecycleEvents.emit("disposed");
-		this.runtime.dispose(error);
```

Comment:
Disposing the runtime might be the only actually interesting thing in here (and it moves to the Container in this PR).  The other calls are disposing proxies, which doesn't seem to do too much.

Reply:
it should def be deprecated and removed from the interface, and only stay on the implementation if we need it.